src/components/PassageDetail/PassageDetailMarkVerses.tsx

```diff
@@ -319,19 +319,27 @@ export function PassageDetailMarkVerses({ width }: MarkVersesProps) {
           const regions = JSON.stringify(segs);
           segmentsRef.current = JSON.stringify({ regions });
         }
+        // update all three segment types: verse, transcription, backtranslation
         projectSegmentSave({
           media,
           segments: updateSegments(
-            NamedRegions.Verse,
-            media.attributes?.segments,
+            NamedRegions.BackTranslation,
+            updateSegments(
+              NamedRegions.Transcription,
+              updateSegments(
+                NamedRegions.Verse,
+                media.attributes?.segments,
+                segmentsRef.current
+              ),
+              segmentsRef.current
+            ),
             segmentsRef.current
```

Comment:
Is this overwriting any that they may have already set in BT and Transcription?

Reply:
Yes, but I guess the questions is what should happen when they change verse mark up after creating back translation?

Reply:
I only update back translation step if there are no recordings and warn the user otherwise.